frontend/src/component/common/SegmentItem/SegmentItem.styles.ts

```diff
@@ -28,6 +28,9 @@ export const useStyles = makeStyles()(theme => ({
     },
     accordionRoot: {
         transition: 'all 0.1s ease',
+        '&:before': {
```

Comment:
same fix we already had for material ui adding background color line in accordion